src/scripts/reducers/index.js

```diff
@@ -2,12 +2,12 @@
 
 import { combineReducers } from 'redux'
 
-import video from 'reducers/VideoReducer'
+import selectedVideo from 'reducers/SelectedVideoReducer'
 import user from 'reducers/UserReducer'
-import uploads from 'reducers/UploadReducer'
+import videos from 'reducers/VideosReducer'
 
 export default combineReducers({
-  selectedVideo: video,
-  user,
-  uploads
+  selectedVideo: selectedVideo,
```

Comment:
Can this be changed to `selectedVideoId`? The way it is currently named, it is implied this this is a video record not a number/string